responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -96,12 +72,17 @@ def __init__(self, model, train, test, target_column,
         :type task_type: str
         :param categorical_features: The categorical feature names.
         :type categorical_features: list[str]
+        :param metadata_columns: The set of columns that are not passed
```

Comment:
Can you elaborate more on what other analysis is intended?

Reply:
same as in the other comment

Reply:
should this be analysis instead of analyses?

Reply:
No, intended to be plural. Added some more context.